examples/creatives/simid_overlay.html

```diff
@@ -10,7 +10,11 @@
 	<button id="request_full_screen">Request Full Screen</button><br>
 	<button id="fatal_error">Fatal Error</button><br>
 	<button id="request_skip">Request Skip</button><br>
-	<button id="request_stop">Request Stop</button><br>
+  <button id="request_stop">Request Stop</button><br>
```

Comment:
Do you need to use the br tag here? I know the other buttons also use it so might be a good idea to refactor those as well

Reply:
I can get rid of the br tag on line 13, but the layout of the creative is to have the buttons in a vertical line, in that case the br tags are necessary